### ISSUE TITLE
ML-Benchmark prediction submission: CNRM-UNeT

### ISSUE THREAD

**Issue by antoinedoury** [WCRP-CORDEX/ml-benchmark#33]
### Institution ID

CNRM

### Institution Name

Centre National de Recherche Météorologique

### Emulator identifier (`emulator_id`)

CNRM-UNeT

### Emulator Description

CNRM-UNeT described in Doury et al 2023, 2024. The architecture is based on a classical UNeT with 2 main differences. The bottom of the U, goes until a flat array, allowing to introduce a 1D vector of inputs (cf below). The expansion path is longer than the encoding path. 
The emulator is trained using the entire set of predictors except the orography. The predictors are standardized such that each input map is normalized (i.e centered in 0 and variance 1) and for each day each variable, the mean and standard deviation used for normalization are given to the network as the 1D inputs. No standardization for outputs.
The network uses the Adam optimizer, a scheduler for the reduction of the learning rate during the training after 4 epochs without improvements. The training stops after 15 epochs with no improvements. 
The loss function for precipitation is the asymmetric loss function described in Doury et al 2024.

<img width="2434" height="1635" alt="Image" src="https://github.com/user-attachments/assets/b611748a-b3cc-451b-b0a8-72489856bafd" />

Doury, A., Somot, S., Gadat, S. et al. Regional climate model emulator based on deep learning: concept and first evaluation of a novel hybrid downscaling approach. Clim Dyn 60, 1751–1779 (2023). https://doi.org/10.1007/s00382-022-06343-9

Doury, A., Somot, S. & Gadat, S. On the suitability of a convolutional neural network based RCM-emulator for fine spatio-temporal precipitation. Clim Dyn 62, 8587–8613 (2024). https://doi.org/10.1007/s00382-024-07350-8

### Hardware and Training Details

Hardware : GPU A30, 24GB. 
Training : 1min40 /epochs, 35 epochs for precipitation so about 1hrs, 70 epochs for tasmax so about 2hrs. 
Inference : 30secondes

### Stochastic/Probabilistic Output

no

### Reference URL

 https://doi.org/10.1007/s00382-022-06343-9 ; https://doi.org/10.1007/s00382-024-07350-8

### Repository for Reproducibility

https://github.com/antoinedoury/RCM-Emulator
